rajawali/src/main/java/rajawali/renderer/RajawaliRenderer.java

```diff
@@ -1310,4 +1410,63 @@ public void setUsesCoverageAaAll(boolean usesCoverageAa) {
 			}
 		}
 	}
+
+    @SuppressLint("HandlerLeak")
+    private final Handler mLoaderHandler = new Handler(Looper.getMainLooper()) {
+
+        @Override
+        public void handleMessage(Message msg) {
+
+            final int id = msg.arg2;
+            final ALoader loader = mLoaderThreads.get(id).mLoader;
+            final IAsyncLoaderCallback callback = mLoaderCallbacks.get(id);
+
+            mLoaderThreads.remove(id);
+            mLoaderCallbacks.remove(id);
```

Comment:
If the loader crashes, the callback will never be released. See the above comment.